.github/workflows/pkg.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Unit Tests
         run: go test -race ./... -coverpkg=./... -coverprofile=pkg_coverage.out
       
+      - name: Fuzz Tests
+        run: go test github.com/smartcontractkit/chainlink-common/pkg/loop/internal -fuzz . -fuzztime 12s
```

Comment:
Maybe no harm to leave in as-is, but this quarter @vyzaldysanchez is working on a standard way to fuzz and starting with this repo.

Reply:
Ok, I can sync with him.  Maybe we get it into `Next` for now, and then I can see what the status is.  If it's close, I can wait or help, otherwise, he can get rid of this once he's ready :)

Reply:
It's not going to be ready yet, he said I could check this in and he'll merge into what he's doing when it is.